nuxt.config.js

```diff
@@ -45,6 +45,13 @@ module.exports = {
         href: env.API_URL,
         crossorigin: 'anonymous'
       },
+      {
+        rel: 'stylesheet preload',
+        as: 'style',
+        href: 'https://use.fontawesome.com/releases/v5.0.10/css/all.css',
```

Comment:
we could use ENV variable with default value for Font Awesome version WDYT @FieryCod ?

Reply:
I don't know whether I understand you correctly. You really want to create ENV variable for that? What would be the purpose of it? Which situation it might handle?

Reply:
teoretically it could be possible to use new version without redeploying application etc., but yeah, I think that's too much for this project

Reply:
Ok I see your reasoning but I think that is too much as you said ;D